x/ibc/05-port/types/module.go

```diff
@@ -0,0 +1,75 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/cosmos-sdk/x/capability"
+	channelexported "github.com/cosmos/cosmos-sdk/x/ibc/04-channel/exported"
+	channeltypes "github.com/cosmos/cosmos-sdk/x/ibc/04-channel/types"
+)
+
+type IBCModule interface {
+	OnChanOpenInit(
+		ctx sdk.Context,
+		order channelexported.Order,
+		connectionHops []string,
+		portID string,
+		channelID string,
+		chanCap capability.Capability,
+		counterParty channeltypes.Counterparty,
+		version string,
+	) error
+
+	OnChanOpenTry(
+		ctx sdk.Context,
+		order channelexported.Order,
+		connectionHops []string,
+		portID,
+		channelID string,
+		portCap capability.Capability,
```

Comment:
We shouldn't need to pass the port capability to the module

Reply:
(we *do* need to pass the channel capability)

Reply:
whoops misnamed